src/go/rpk/pkg/cli/cmd/redpanda/start.go

```diff
@@ -505,7 +509,15 @@ func NewStartCommand(fs afero.Fs, launcher rp.Launcher) *cobra.Command {
 		"Enable overprovisioning",
 	)
 	command.Flags().BoolVar(&sFlags.unsafeBypassFsync, unsafeBypassFsyncFlag, false, "Enable unsafe-bypass-fsync")
-	command.Flags().BoolVar(&sandbox, sandboxFlag, false, "Sandbox mode, sets the following flags: --overprovisioned --smp 1 --node-id 0 --reserveMemory 0M --check=false --unsafe-bypass-fsync=true")
+	command.Flags().StringVar(
+		&sandbox,
+		sandboxFlag,
+		"",
+		"Sandbox mode, sets relevant properties for local development or test container usage; use --sandbox help for more info",
```

Comment:
cc @emaxerrno what should we say here, local development or something else?

Reply:
```

Sandbox mode, sets well-known configuration properties for development or test environments; use --sandbox help ...


```

Reply:
Done :+1: